dradis/dradis-nexpose
# Evidence template

### Spec

Right now, importing a file through the NeXpose plugin creates Evidences without using any template. This makes exporting Evidences difficult.

**Proposed solution**

One solution to this is to create Evidences through the TemplateService.

### How to test
1. Upload a sample Nexpose xml  `spec/fixtures/files/full.xml`
2. Check that evidences are created with the `Details` header.
3. Edit the template in the Plugins tab.
4. Upload the sample xml again.
5. Assert that the created evidences reflected the changes on the template.
